### PR TITLE
feat(saved-searches): Add create saved search button to sidebar

### PR DESCRIPTION
**Pull request by malwilley**
image

### CODE REVIEW

**Comment by evanpurkhiser** [comment]
Could you toss a screenshot in?

File: static/app/views/issueList/savedIssueSearches.tsx [hunk 1]
```diff
@@ -174,13 +224,20 @@ const StyledSidebar = styled('aside')`
   }
 `;
 
-const Heading = styled('h2')`
+const HeadingContainer = styled('div')`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 38px;
```
Comment (evanpurkhiser) [inline static/app/views/issueList/savedIssueSearches.tsx:231]: do we have to specify a height?

**Review by evanpurkhiser — APPROVED** [review]

File: static/app/views/issueList/savedIssueSearches.tsx (hunk 1 above)
Comment (malwilley) [reply static/app/views/issueList/savedIssueSearches.tsx:231]: am specifying a height here to make sure each heading is the same height - the saved searches header has the `+` button which makes it slightly larger and thus further away from the content below

Comment (evanpurkhiser) [reply static/app/views/issueList/savedIssueSearches.tsx:231]: got you

File: static/app/views/issueList/filters.tsx [hunk 2]
```diff
@@ -46,15 +46,17 @@ function IssueListFilters({
...
+          organization.features.includes('issue-list-saved-searches-v2')
+            ? []
+            : [
+                makePinSearchAction({sort, pinnedSearch, location}),
+                makeSaveSearchAction({
+                  sort,
+                  disabled: !organization.access.includes('org:write'),
+                }),
+              ]
+        }
```
Comment (evanpurkhiser) [inline static/app/views/issueList/filters.tsx:59]: why did this change?

**Review by evanpurkhiser — APPROVED** [review]

File: static/app/views/issueList/filters.tsx (hunk 2 above)
Comment (malwilley) [reply static/app/views/issueList/filters.tsx:59]: Hides the `+` button in the search bar if you've got the feature flag on

**Pull request merged**